bundle.mustache

```diff
@@ -10,31 +13,45 @@ URL: {{{PROJECTURL}}}
 {{#each SOURCES}}
 {{{.}}}
 {{/each}}
+%if 0%{?scl:1}
+BuildRequires: %{?scl_prefix_nodejs}build
+BuildRequires: %{?scl_prefix_nodejs}npm
+%else
 BuildRequires: nodejs-packaging
+%endif
 BuildArch: noarch
 ExclusiveArch: %{nodejs_arches} noarch
 
-Provides: npm(%{npm_name}) = %{version}
+Provides: %{?scl_prefix_nodejs}npm(%{npm_name}) = %{version}
 {{#each PROVIDES}}
-Provides: bundled(npm({{{name}}})) = {{{version}}}
+Provides: bundled(%{?scl_prefix_nodejs}npm({{{name}}})) = {{{version}}}
 {{/each}}
 AutoReq: no
 AutoProv: no
```

Comment:
I'd like to revisit this within SCL. The script appears to handle bundled dependencies which would remove the need for large spec files. It also looks like they're slightly different than what we use:
```python
    if bundled:
        value = "bundled(nodejs-%s) = %s" % (name, version)
    else:
        value = "rh-nodejs10-npm(%s) = %s" % (name, version)
```

Reply:
I am not following what the ask is.

Reply:
Currently we're not using the AutoReq and AutoProv mechanism in RPMs but the scripts in the SCL appear to be able to automatically generate those `Provides` statements. They are slightly different than what we output now so the question is whether that's a problem. Currently I don't know the answer so I'm hoping you have some :)

Reply:
From my testing, turning these on reseults in the following provides:

```
scl-package(tfm)
tfm-nodejs-history = 4.9.0-1.el7
```

Reply:
Worth noting, rh-nodejs10 packages with bundled dependencies don't include the SCL (which I suppose makes some sense):

https://centos.pkgs.org/7/centos-sclo-rh-testing-x86_64/rh-nodejs10-npm-6.4.1-10.10.0.2.el7.x86_64.rpm.html